src/components/pages/Car/Cars.jsx

```diff
@@ -1,9 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import './Cars.css';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchCars, selectCars, selectCarsStatus,
+} from '../../../redux/cars/carsSlice';
 
-const Cars = () => (
-  <>
-    <h1>List of Cars</h1>
-  </>
-);
+const Cars = () => {
+  const dispatch = useDispatch();
+  const cars = useSelector(selectCars);
+  const status = useSelector(selectCarsStatus);
+
+  useEffect(() => {
+    dispatch(fetchCars());
+  }, [dispatch]);
+
+  return (
+    <div className="cars-container">
+      <h1>List of Cars</h1>
+      {status === 'loading' && <p>Loading...</p>}
+      {status === 'failed' && <p>Error loading cars.</p>}
```

Comment:
OPTIONAL CHANGES
On loading the dashboard, as a new user who hasn't created any cars, it displays "Error loading cars." Why not make a condition to tell the user they haven't created any cars? for example, for "You have not created any cars." 

![Screenshot 2023-12-23 at 14 25 59](https://github.com/ahmidou09/cruise-fleet-frontend/assets/88312845/e5caf2db-b059-47ae-bea5-7a54ba81575a)

Reply:
The message "Error loading cars" typically indicates a technical server issue, not the absence of created cars.